polkadot/erasure-coding/src/lib.rs

```diff
@@ -449,3 +455,56 @@ mod tests {
 		}
 	}
 }
+
+/// Helper functions for feature-aware erasure coding
+pub mod feature_aware {
+	use super::*;
+	use polkadot_node_primitives::AvailableData;
+	
+	/// Check if FastErasureCoding feature is enabled in node features
```

Comment:
Where can I see some benchmarks how it performs against current implementation ?